src/main/perf/LineFileDocs.java

```diff
@@ -165,9 +167,10 @@ private void readDocs() throws Exception {
           throw new RuntimeException("expected " + length + " document bytes but read " + x);
         }
         buffer.position(0);
-        queue.put(new LineFileDoc(buffer, readVector(count)));
+        queue.put(new LineFileDoc(buffer, readVector(docCountPerBlock), idBase));
       }
     } else {
```

Comment:
Maybe add a comment to this line lol (to know what the `else` case is handling).

Reply:
OK for sure!